src/js/brandColor/brandColor.types.ts

```diff
@@ -0,0 +1,130 @@
+export interface BrandColor {
+  /** white/white000: #ffffff */
```

Comment:
These comments align with the Figma naming that engineers see in Figma and on hover of the object will also see the hex value 


https://github.com/MetaMask/design-tokens/assets/8112138/a2c6ddc7-a949-4806-af49-952472c6f11a